enso/visualize/facets.py

```diff
@@ -47,26 +48,31 @@ def visualize(
         if isinstance(lines, (tuple, list)):
             results['key'] = results[lines].apply(lambda x: ','.join(x), axis=1)
             lines = 'key'
-
-        n_lines = len(results[lines].unique())
-        rc = {'lines.linewidth': 1, 'lines.markersize': 1}
-        sns.set_context("paper", rc=rc)
-        grid = sns.FacetGrid(
-            results,
-            col=x_tile,
-            row=y_tile,
-            hue=lines,
-            size=5,
-            legend_out=False,
-            margin_titles=True,
-            sharey=False,
-            sharex=False,
-            palette=sns.color_palette("hls", n_lines)
-        )
-        grid = grid.map(sns.pointplot, x_axis, y_axis)
-        colors = sns.color_palette("hls", n_lines).as_hex()[:len(grid.hue_names)]
-        handles = [patches.Patch(color=col, label=label) for col, label in zip(colors, grid.hue_names)]
-        plt.legend(handles=handles)
-        plt.tight_layout()
+        datasets = np.unique(results.Dataset)
```

Comment:
Looks great! 

In the current state we do lose some small amount of functionality that we currently support re: selecting what columns represent the x and y tiling, lines, etc. See: https://github.com/IndicoDataSolutions/Enso/blob/master/enso/config.py#L115

This is low priority though.  I'm personally OK with merging this as is, but also think it should be not too time consuming to add that support back in.  Will let that be your call!